charts/clearml/values.yaml

```diff
@@ -291,6 +291,27 @@ agentGroups:
 
     affinity: {}
 
+# This agent will spawn queued experiments in new pods, a good use case is to combine this with
+# GPU autoscaling nodes.
+# https://github.com/allegroai/clearml-agent/tree/master/docker/k8s-glue
+agentk8sglue:
+  enabled: true
```

Comment:
@nielstenboom I think this should be `false` by default 🙂

Reply:
Ah yes of course, thanks! 👍